src/amo/pages/AddonVersions/styles.scss

```diff
@@ -12,15 +12,16 @@
   .CardList ul {
     li {
       margin-bottom: 0;
-    }
-
-    .AddonVersions-warning {
-      padding: $padding-page;
       padding-bottom: 0;
 
+      &:last-of-type {
+        padding-bottom: $padding-page;
+      }
+
       @include respond-to(large) {
```

Comment:
You could have moved the `@include` in the block above, no?

Reply:
I tried that, and got a stylelint issue with too much nesting. :-(

Reply:
ah :(